src/calc/__init__.py

```diff
@@ -11,8 +12,9 @@
     "get_sum_of_strings",
     "get_square",
     "get_squares",
+    "remove_duplicate_lists"
 ]
 
 from calc.func import (get_factorial, get_fibonacci_number,
                        get_fibonacci_number_nr, get_square, get_squares,
-                       get_sum_of_strings)
+                       get_sum_of_strings, remove_duplicate_lists)
```

Comment:
I copied the wrong path last time, sorry.
`calc` package is not affected by these changes at all.